conda/exports.py

```diff
@@ -104,8 +104,12 @@
 get_rc_urls = lambda: list(conda.base.context.context.channels)
 get_local_urls = lambda: list(get_conda_build_local_url()) or []
 load_condarc = lambda fn: conda.base.context.reset_context([fn])
-from .exceptions import PaddingError  # NOQA
+from .exceptions import PaddingError, LinkError, CondaOSError, CondaFileNotFoundError  # NOQA
 PaddingError = PaddingError
+LinkError = LinkError
+CondaOSError = CondaOSError
+# clobbering Python's FileNotFoundError - be sure to use module namespacing
```

Comment:
Oh, right, `FileNotFound` is a python3--but not python2--thing.  Should we rename the exception to `PathNotFoundError` instead?  Now would be a good time to do it if you agree.